client/src/api/user-api.js

```diff
@@ -0,0 +1,20 @@
+import api from "/api";
+
+async function createUser(auth0_id, controller) {
```

Comment:
stick with camelCase for variable names

Reply:
not sure what you're talking about here, all of the api function names have camel case

Reply:
auth0_id from the parameters is not in camelcase

The object key is fine, but not the value